app/src/main/java/org/simple/clinic/instantsearch/InstantSearchUpdate.kt

```diff
@@ -58,7 +58,12 @@ class InstantSearchUpdate : Update<InstantSearchModel, InstantSearchEvent, Insta
   private fun searchResultsLoaded(model: InstantSearchModel, event: SearchResultsLoaded): Next<InstantSearchModel, InstantSearchEffect> {
     if (!model.hasSearchQuery) return noChange()
 
-    return dispatch(ShowPatientSearchResults(event.patientsSearchResults, model.facility!!))
+    val effect = if (event.patientsSearchResults.isNotEmpty())
+      ShowPatientSearchResults(event.patientsSearchResults, model.facility!!)
```

Comment:
Search results should be a part of the model and rendered via the UI renderer. Why are we loading them via effects? 🤔

Reply:
We discussed this before since we will be replacing these lists with paging data, we cannot place them in models. So for that reason, I had to switch to using effects for handling lists and error cases. Should we change it?

https://simpledotorg.slack.com/archives/C01FN9GCNAJ/p1609295944003000

Reply:
Ah, ok. I guess that makes sense.

Reply:
Makes sense, let's leave it as is.